newIDE/app/public/external/jfxr/README.md

```diff
@@ -0,0 +1,7 @@
+This folder contains sources to embed Jfxr editor (https://github.com/ttencate/jfxr) so that it can
+be used directly from GDevelop to edit sound effects.
+
+Jfxr sources are downloaded by `import-zipped-editor.js` script. They are raw, unchanged sources
+of the Jfxr editor build. Sources will be stored in `Jfxr-editor` folder.
+See `jfxr-main.js` for the code running the editor.
+See `ModalWindow.js` and `LocalPiskelBridge.js` files for the bridge that open the Window and pass data from GDevelop to Jfxr.
```

Comment:
Thanks for thinking about the documentation :) Typo: this should be `LocalJfxrBridge.js` here.

Reply:
good catch! Fixed now :)